shared/k8s/debug.go

```diff
@@ -2,18 +2,34 @@ package k8s
 
 import (
 	"context"
+	"fmt"
+	"os"
 
+	"github.com/pokt-network/pocket/logger"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
 //nolint:gosec // G101 Not a credential
-const privateKeysSecretResourceName = "v1-localnet-validators-private-keys"
+const privateKeysSecretResourceName = "validators-private-keys"
+
+var CurrentNamespace = ""
+
+func init() {
+	var err error
+	CurrentNamespace, err = getNamespace()
+	if err != nil {
+		logger.Global.Err(err).Msg("could not get namespace, using default")
+		CurrentNamespace = "default"
+	}
+
+	logger.Global.Info().Str("namespace", CurrentNamespace).Msg("using namespace")
+}
 
 func FetchValidatorPrivateKeys(clientset *kubernetes.Clientset) (map[string]string, error) {
 	validatorKeysMap := make(map[string]string)
 
-	privateKeysSecret, err := clientset.CoreV1().Secrets("default").Get(context.TODO(), privateKeysSecretResourceName, metav1.GetOptions{})
+	privateKeysSecret, err := clientset.CoreV1().Secrets(CurrentNamespace).Get(context.TODO(), privateKeysSecretResourceName, metav1.GetOptions{})
```

Comment:
This was a change that was going to be necessary for later Byzantine test network stuff so good to see it going in anyway! 🙌